package.json

```diff
@@ -18,7 +18,7 @@
       "\\.svg": "<rootDir>/__mocks__/svgMock.js"
     },
     "transformIgnorePatterns": [
-      "node_modules/(?!(jest-)?react-native|react-clone-referenced-element|@react-native-community|expo(nent)?|@expo(nent)?/.*|react-navigation|@react-navigation/.*|@unimodules/.*|unimodules|sentry-expo|native-base|@sentry/.*|matomo-tracker-react-native|ramda)"
+      "node_modules/(?!(jest-)?@?react-native|react-clone-referenced-element|@react-native-community|expo(nent)?|@expo(nent)?/.*|react-navigation|@react-navigation/.*|@unimodules/.*|unimodules|sentry-expo|native-base|@sentry/.*|matomo-tracker-react-native|ramda)"
```

Comment:
did you read this somewhere? how come that this is now needed?

Reply:
react native types where changed around and not having this leads to errors because of type declarations being present in places where there previously where none.